src/Juvix/Compiler/Concrete/Data/Scope.hs

```diff
@@ -105,3 +105,9 @@ emptyScope absPath =
       _scopeTopModules = mempty,
       _scopeLocalSymbols = mempty
     }
+
+newtype Reserved = Reserved
```

Comment:
This type has returned 👻 - perhaps cherry-pick the commit that does the formatting onto a new branch?

Reply:
I cherry picked the three relevant commits on a new branch based on main, ty!